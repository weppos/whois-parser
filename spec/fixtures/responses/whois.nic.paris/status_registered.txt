
Domain Name: bienvenue.paris
Registry Domain ID: DOM000000021725-PARIS
Registrar WHOIS Server: whois.nameshield.net
Registrar URL: http://www.nameshield.net
Updated Date: 2020-04-26T22:20:43Z
Creation Date: 2014-06-03T13:05:37Z
Registry Expiry Date: 2022-06-03T13:05:37Z
Registrar: NAMESHIELD
Registrar IANA ID: 1251
Registrar Abuse Contact Email: abuse@nameshield.net
Registrar Abuse Contact Phone: +33 2 41 18 28 28
Domain Status: clientDeleteProhibited  https://icann.org/epp#clientDeleteProhibited
Domain Status: clientTransferProhibited  https://icann.org/epp#clientTransferProhibited
Registry Registrant ID: REDACTED FOR PRIVACY
Registrant Name: REDACTED FOR PRIVACY
Registrant Organization: VILLE DE PARIS
Registrant Street: REDACTED FOR PRIVACY
Registrant City: REDACTED FOR PRIVACY
Registrant State/Province: N/C
Registrant Postal Code: REDACTED FOR PRIVACY
Registrant Country: FR
Registrant Phone: REDACTED FOR PRIVACY
Registrant Phone Ext: REDACTED FOR PRIVACY
Registrant Fax: REDACTED FOR PRIVACY
Registrant Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Registry Admin ID: REDACTED FOR PRIVACY
Admin Name: REDACTED FOR PRIVACY
Admin Organization: REDACTED FOR PRIVACY
Admin Street: REDACTED FOR PRIVACY
Admin City: REDACTED FOR PRIVACY
Admin State/Province: REDACTED FOR PRIVACY
Admin Postal Code: REDACTED FOR PRIVACY
Admin Country: REDACTED FOR PRIVACY
Admin Phone: REDACTED FOR PRIVACY
Admin Phone Ext: REDACTED FOR PRIVACY
Admin Fax: REDACTED FOR PRIVACY
Admin Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Registry Tech ID: REDACTED FOR PRIVACY
Tech Name: REDACTED FOR PRIVACY
Tech Organization: REDACTED FOR PRIVACY
Tech Street: REDACTED FOR PRIVACY
Tech City: REDACTED FOR PRIVACY
Tech State/Province: REDACTED FOR PRIVACY
Tech Postal Code: REDACTED FOR PRIVACY
Tech Country: REDACTED FOR PRIVACY
Tech Phone: REDACTED FOR PRIVACY
Tech Fax: REDACTED FOR PRIVACY
Tech Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Name Server: mdpdns01.paris.fr
Name Server: mdpdns02.paris.fr
Name Server: ns2.perf1.fr
Name Server: ns3.perf1.com
Name Server: ns3.perf1.eu
Name Server: ns4.perf1.de
Name Server: ns5.perf1.asia
DNSSEC: unsigned
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2021-06-09T15:46:09Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

Rights restricted by copyright.

If you wish to access the protected data, please visit our website at the following address: www.bienvenue.paris

Terms of Use : City of Paris's database is protected by the provisions of the French Law of the 1st of July 1998
transposing the European Directive of the 11th of March 1996 on database legal protection in the Intellectual Property Code.

City of Paris is the only entity to hold the producing rights of the databases as substantial investment, quantitative
as much as qualitative, that were used to achieve this database.
Complete or partial reproduction and/or use of City of Paris's database without the express permission of City of Paris
is strictly forbidden.

Any breach to these rules can carry penal sanctions as counterfeit, without prejudice of a potential damages request from City of Paris,
holder of these rights. City of Paris's database user commits to use the published data according to the laws
and rules in effect. Moreover, the user is bound to respect the French Data Protection Law.
The infringement of the French Data Protection Law can carry penal sanctions. As the user will access personal information,
the user must refrain from collecting or misuse this information. In a more general way, the user must refrain from any act likely
to infringe the privacy or reputation of individuals.

