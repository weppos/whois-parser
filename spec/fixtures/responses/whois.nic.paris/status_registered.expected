#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2014-06-03T13:05:37Z}

#updated_on
  %s %CLASS{time}
  %s %TIME{2020-04-26T22:20:43Z}

#expires_on
  %s %CLASS{time}
  %s %TIME{2022-06-03T13:05:37Z}


#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].name          == "REDACTED FOR PRIVACY"
  %s[0].organization  == "VILLE DE PARIS"
  %s[0].address       == "REDACTED FOR PRIVACY"
  %s[0].city          == "REDACTED FOR PRIVACY"
  %s[0].zip           == "REDACTED FOR PRIVACY"
  %s[0].state         == "N/C"
  %s[0].country       == nil
  %s[0].country_code  == "FR"
  %s[0].phone         == "REDACTED FOR PRIVACY ext: REDACTED FOR PRIVACY"
  %s[0].fax           == "REDACTED FOR PRIVACY"
  %s[0].email         == "Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."
  %s[0].updated_on    == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type         == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].name          == "REDACTED FOR PRIVACY"
  %s[0].organization  == "REDACTED FOR PRIVACY"
  %s[0].address       == "REDACTED FOR PRIVACY"
  %s[0].city          == "REDACTED FOR PRIVACY"
  %s[0].zip           == "REDACTED FOR PRIVACY"
  %s[0].state         == "REDACTED FOR PRIVACY"
  %s[0].country       == nil
  %s[0].country_code  == "REDACTED FOR PRIVACY"
  %s[0].phone         == "REDACTED FOR PRIVACY ext: REDACTED FOR PRIVACY"
  %s[0].fax           == "REDACTED FOR PRIVACY"
  %s[0].email         == "Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].name          == "REDACTED FOR PRIVACY"
  %s[0].organization  == "REDACTED FOR PRIVACY"
  %s[0].address       == "REDACTED FOR PRIVACY"
  %s[0].city          == "REDACTED FOR PRIVACY"
  %s[0].zip           == "REDACTED FOR PRIVACY"
  %s[0].state         == "REDACTED FOR PRIVACY"
  %s[0].country       == nil
  %s[0].country_code  == "REDACTED FOR PRIVACY"
  %s[0].phone         == "REDACTED FOR PRIVACY"
  %s[0].fax           == "REDACTED FOR PRIVACY"
  %s[0].email         == "Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{7}
  %s[0] %CLASS{nameserver}
  %s[0].name == "mdpdns01.paris.fr"
  %s[1] %CLASS{nameserver}
  %s[1].name == "mdpdns02.paris.fr"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns2.perf1.fr"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns3.perf1.com"
  %s[4] %CLASS{nameserver}
  %s[4].name == "ns3.perf1.eu"
  %s[5] %CLASS{nameserver}
  %s[5].name == "ns4.perf1.de"
  %s[6] %CLASS{nameserver}
  %s[6].name == "ns5.perf1.asia"
